Sources/Mistica/Components/Feedback/FeedbackView.swift

```diff
@@ -290,32 +322,86 @@ private extension FeedbackView {
             scrollStackView.stackView.insertArrangedSubview(icon, at: 0)
         case .animation(let animation):
             animatedIcon.animation = animation
-            let width = animation.bounds.width
-            let height = animation.bounds.height
-            NSLayoutConstraint.activate([
-                animatedIcon.widthAnchor.constraint(equalToConstant: width),
-                animatedIcon.heightAnchor.constraint(equalToConstant: height)
-            ])
-            // To center animations that are not 1:1 squares
-            let leftMargin = (width - height) / 2
-            animatedIcon.transform = CGAffineTransform(translationX: -leftMargin, y: 0)
             scrollStackView.stackView.insertArrangedSubview(animatedIcon, at: 0)
         }
     }
 
     func prepareAnimation() {
-        guard style.shouldAnimate else { return }
+        // Initial state
+        func prepare(view: UIView) {
+            view.alpha = Constants.Animation.Opacity.initial
+            view.transform = CGAffineTransform(
+                translationX: 0,
+                y: Constants.Animation.Offset.initial
+            )
+        }
+        // Final state
+        func animation(for view: UIView) -> () -> Void {
+            {
+                view.alpha = Constants.Animation.Opacity.final
+                view.transform = CGAffineTransform(
+                    translationX: 0,
+                    y: Constants.Animation.Offset.final
+                )
+            }
+        }
+
+        guard UIView.areAnimationsEnabled, style.shouldAnimate else { return }
         animationFired = false
-        contentContainerStackView.alpha = 0
-        contentContainerStackView.transform = CGAffineTransform(translationX: 0, y: 20)
-        animator.addAnimations {
-            self.contentContainerStackView.alpha = 1
-            self.contentContainerStackView.transform = CGAffineTransform(translationX: 0, y: 0)
+
+        // Views that should animate
+        var views = [UIView]()
+        if title.isEmpty == false {
+            views.append(titleLabel)
+        }
+        if let subtitle, subtitle.isEmpty == false {
+            views.append(subtitleLabel)
+        }
+        if let errorReference, errorReference.isEmpty == false {
+            views.append(errorReferenceLabel)
+        }
```

Comment:
This can be simplified by using the `!` operator
```suggestion
        if !title.isEmpty {
            views.append(titleLabel)
        }
        if let subtitle, !subtitle.isEmpty {
            views.append(subtitleLabel)
        }
        if let errorReference, !errorReference.isEmpty {
            views.append(errorReferenceLabel)
        }
```

Reply:
I'm not a big fan of inverting booleans as I consider it is less readable.